h2o-genmodel/build.gradle

```diff
@@ -31,5 +31,6 @@ dependencies {
   compile("ai.h2o:deepwater-backend-api:1.0.4") {
     exclude group: 'org.slf4j'
   }
+  compile project(":h2o-genmodel-ext-xgboost")
```

Comment:
we cannot have this dependency, gen-model needs to be independent of xgboost

Reply:
This is a leftover as well. There is no need to have this dependency. Removed. Definitely. No butts about it :+1: :D